### ISSUE TITLE
Typing indicator displays "user and 0 more are typing..."

### ISSUE THREAD

**Issue by liamnichols** [GetStream/stream-chat-swift#2751]
## What did you do?

Integrated the StreamChatUI SDK and presented a `ChatChannelVC` instance for a direct message channel created using `ChatClient.channelController(createDirectMessageChannelWith:extraData:)`

## What did you expect to happen?

When the other member types, you'd expect to see "{user} is typing..." 

<img src="https://github.com/GetStream/stream-chat-swift/assets/482871/aed7118f-69b5-414d-8ff2-2fbb09e27111" width="187.5" />

## What happened instead?

You see "{user} and 0 more are typing..."

## GetStream Environment
**GetStream Chat version:** 4.35.2
**GetStream Chat frameworks:** StreamChat, StreamChatUI
**iOS version:** 16.4
**Swift version:** 5.8
**Xcode version:** 14.3.1
**Device:** iPhone 14

**Comment by liamnichols** [comment]
I worked around it for now with this subclass:

```swift
final class ChatMessageListVC: StreamChatUI.ChatMessageListVC {
    override func showTypingIndicator(typingUsers: [ChatUser]) {
        // If the typingUsers count is anything other than 1, use the super implementation..
        // We're just fixing a bug for when there is 1 user
        guard typingUsers.count == 1, let name = typingUsers.first?.name else {
            super.showTypingIndicator(typingUsers: typingUsers)
            return
        }

        guard isTypingEventsEnabled else { return }
        typingIndicatorView.content = name + " " + R.string.streamChatUI.channelItemTypingSingular()
        typingIndicatorView.isHidden = false
    }
}
```

**Comment by nuno-vieira** [comment]
Hi @liamnichols,

I can't reproduce this bug 🤔 Did you change the localization of your app? Because this logic is part of the Localization of Stream. If there is only 1 user typing, the message should display correctly.

Best,
Nuno

**Comment by nuno-vieira** [comment]
@liamnichols We are going to close this one for now, if more people report this, we will look into it, but it does seem the the issue is not on our side in this case 🤔 

Best,
Nuno

**Comment by liamnichols** [comment]
Ah @nuno-vieira, thanks for pointing that out. Yes, I uploaded the .stringsdict to a translation platform and re-imported it which has caused `zero` to be removed.

However the reason that `zero` was removed is valid because it is not a supported category for the English language as per the Unicode CLDR language plural rules which many platforms are following: https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html

For this reason, it's not possible to rely on using the `zero` category in English as well as a series of other languages for this kind of behaviour. The only way around this is to have a dedicated phrase for this scenario like I apply in my workaround.
